docs/assets/js/tabs.js

```diff
@@ -1,52 +1,259 @@
-import { Tabs } from 'govuk-frontend';
-
-/**
- * @param {DOMNode} container - Parent element containing Gov UK tab HTML
- */
-export function init( container ) {
-  const tabs = container.querySelectorAll( '[data-module="tabs"]' );
-
-  if ( tabs && tabs.length > 0 ) {
-    for ( let i = 0; i < tabs.length; i++ ) {
-      const tab = tabs[i];
-      new Tabs( tab ).init();
+const BASE_CLASS = 'm-tabs';
+
+function Tabs() {
+  const tablist = document.querySelectorAll('[role="m-tabs__list"]')[0];
```

Comment:
Search for the tab class inside the base class, so have something like:

```
const tabsContainerDom = document.querySelector( `.${ BASE_CLASS }` );
const tabsListDom = tabsContainerDom.querySelectorAll( `.${ BASE_CLASS }_list` );
```

Also, both these lookups I'd put in the init method, so:

```
let tabsContainerDom;
let tabsListDom;

function init() {
  const tabsContainerDom = document.querySelector( `.${ BASE_CLASS }` );
  const tabsListDom = tabsContainerDom.querySelectorAll( `.${ BASE_CLASS }_list` );

  return this;
}
```

This is just. so when you create the instance, it creates the internal. variables, but they don't get assigned a value till `init` is called.